test/test_esapi.py

```diff
@@ -4716,6 +4716,21 @@ def test_ref_parent(self):
         gc.collect()
         self.assertEqual(bytes(algb), TPM2_ALG.SHA256.to_bytes(2, "big"))
 
+    def test_double_close(self):
```

Comment:
This test seems to hang for me locally as well.

Reply:
Oh I think I see it. We have a connection to mssim in self.ectx, and use the same information to try and connect to it again, but only one client can be connected at a time.